index.html

```diff
@@ -2203,26 +2203,30 @@ <h3>
           <a>purpose</a>.
         </p>
         <p class="note">
-          For example, an icon with purpose "<a>badge</a>" could be used as a
-          badge or pinned icon that is visually distinct, in color or form,
-          from an application's launch icon. The user agent uses the value of
-          the <a>purpose</a> member as a hint to determine where and how an
-          <a>ImageResource</a> is displayed. Unless declared otherwise by the
-          developer, a user agent can use an icon for <a>any purpose</a>.
+          For example, an icon with purpose "<a>monochrome</a>" could be used
+          as a badge or pinned icon with a solid fill, visually distinct from
+          an application's full color launch icon. The user agent uses the
+          value of the <a>purpose</a> member as a hint to determine where and
+          how an <a>ImageResource</a> is displayed. Unless declared otherwise
+          by the developer, a user agent can use an icon for <a>any
+          purpose</a>.
         </p>
         <p>
           The <dfn>icon purposes</dfn> are as follows:
         </p>
         <dl>
           <dt>
-            <dfn data-lt="badge purpose">badge</dfn>:
+            <dfn data-lt="monochrome purpose">monochrome</dfn>:
           </dt>
           <dd>
-            A user agent can present this icon where space constraints and/or
-            color requirements differ from those of the application icon.
+            A user agent can present this icon where a
+            <a href="#monochrome-icons-and-solid-fills">monochrome icon with a
+            solid fill</a> is needed. The color information in the icon is
+            discarded and only the alpha data is used. The icon can then be
+            used user agent like a mask over any colored background.
```

Comment:
I've tried to limit the usage of the term "mask", but the actual image processing steps is still masking, just using the icon as the mask instead of the image to mask. Because of this, I think rephrasing would hurt readability.